client/src/Components/QuestionsContainer/QuestionsList/QuestionsList.jsx

```diff
@@ -0,0 +1,120 @@
+import React, { useState, useEffect } from "react"
+import { ListGroup, Button } from "react-bootstrap"
+import QuestionListEntry from "./QuestionsListEntry/QuestionsListEntry.jsx"
+import QuestionModal from "./QuestionModal/QuestionModal.jsx"
+import useQA from "../../../contexts/QAContext.jsx"
+import styled from "styled-components"
+import InfiniteScroll from "react-infinite-scroll-component"
+
+const Button1 = styled.button`
+  margin-right: 15px;
+  border-radius: 0px;
+  border: 2px solid black;
+  background-color: white;
+  color: black;
+  padding: 14px 28px;
+  font-size: 16px;
+  cursor: pointer;
+`
+
+const QuestionsList = (props) => {
```

Comment:
You could destructure props if there aren't that many
```suggestion
const QuestionsList = ({ filterBySearch, searchValue }) => {
```